app/assets/javascripts/application.js

```diff
@@ -29,29 +29,36 @@ var ToEndDate = new Date();
 
 ToEndDate.setDate(ToEndDate.getDate()+365);
 
-$('.from_date').datepicker({
+$(".from_date").datepicker({
 
     weekStart: 1,
-    startDate: '01/01/2012',
+    startDate: "01/01/2012",
     endDate: FromEndDate,
     autoclose: true
-})
-    .on('changeDate', function(selected){
+});
+    .on("changeDate", function(selected){
         startDate = new Date(selected.date.valueOf());
         startDate.setDate(startDate.getDate(new Date(selected.date.valueOf())));
-        $('.to_date').datepicker('setStartDate', startDate);
+        $(".to_date").datepicker("setStartDate", startDate);
     });
-$('.to_date')
+$(".to_date")
     .datepicker({
 
         weekStart: 1,
         startDate: startDate,
         endDate: ToEndDate,
         autoclose: true
-    })
-    .on('changeDate', function(selected){
+    });
+    .on("changeDate", function(selected){
```

Comment:
Expected an identifier and instead saw '.'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'on' is not defined.